# Stage estivo 2021 - Cosmologia
Hanno contribuito alla realizzazione di questo notebook: Marco Bortolami, Shahbaz Alvi, Luca Caloni, Serena Giardiello, Caterina Chiocchetta, Matteo Billi, Margherita Lembo, Umberto Natale.

# Carichiamo i moduli necessari
15 min in tutto

In [ ]:
import numpy as np              # modulo che fornisce le funzioni matematiche
import matplotlib.pyplot as plt # modulo che fornisce funzioni per produrre grafici 

Esegui la cella sottostante solo se sei in ambiente Google Drive

In [ ]:
# Autentichiamoci con il nostro account gmail
# Esegui il comando in basso, clicca sul link, autenticati con il tuo indirizzo gmail,
# copia il link fornito dopo l'autenticazione e incollalo nel rettangolo
from google.colab import drive
drive.mount('/content/drive')

Mounted at /content/drive


Di seguito, lavoreremo con dei dati reali di misure di supernovae. I comandi necessari per analizzare e graficare questi dati sono forniti nelle celle che seguono.
Eseguiremo insieme le prime celle, dopodiche' vi verra' chiesto di scrivere da soli i comandi necessari per eseguire alcune operazioni sui dati.

Scrivete il codice mancante nelle celle vuote. Ogni cella vuota e' seguita da una cella nascosta. Se cliccate sul bottone 'Show Text', vi comparira' il contenuto della cella nascosta, che rappresenta la soluzione. Cercate di farvi mostrare il contenuto della soluzione solo se proprio non avete idea di come risolvere il passaggio richiesto. Se avete difficolta', provate prima a chiedere consiglio a noi.

Buon divertimento!

# Carichiamo i dati

In [ ]:
# Usa il link sottostante solo se sei in ambiente Google Drive
path_to_data='/content/drive/Shared drives/Stage-Estivi-a-Fisica-2021-studenti/cognome.nome/cosmologia/gold.dat'
# Sostituire nella stringa di sopra il proprio cognome.nome nello spazio apposito
data=np.loadtxt(path_to_data)
data.shape # Supernovae: redshift (z), modulo di distanza (Md), errore sul modulo di distanza (err_Md) 

(157, 3)

Abbiamo salvato il contenuto del file in un oggetto che e' fatto come una griglia (matrice) di 157 righe e 3 colonne. Possiamo accedere all'elemento ij che occupa lo spazio all'incrocio tra la riga i e la colonna j scrivendo
\begin{equation*}
data[i,j]
\end{equation*}

Quindi, l'elemento i=0, j=0 e' il redshift della prima supernova, cioè
\begin{equation*}
data[0,0]
\end{equation*}

mentre l'elemento i=130, j=1 e' il modulo di distanza della 131-esima supernova, cioè
\begin{equation*}
data[130,1]
\end{equation*}

<figure>
<center>
<img src='https://drive.google.com/uc?export=view&id=1OW0lUuX5X-6wD6oRVG6rTVPM09iMQ4Bp' width="300" height="300">
</center>
</figure>

Nota che iniziamo a contare da 0. Python e' uno dei linguaggi di programmazione che sceglie di indicare il primo elemento di ogni riga o colonna con 0. La prima riga della matrice e' quindi la numero 0, mentre l'ultima e' la numero 156. Allo stesso modo, la prima colonna e' la numero 0, mentre l'ultima e' la numero 2.
    
Nel nostro caso, abbiamo quindi 3 colonne di dati per 157 Supernovae. Per ogni riga, le colonne rappresentano il redshift z della supernova, il modulo di distanza M-m (un indicatore della distanza dell'oggetto) e l'errore sul modulo di distanza err_Md. 

Per comodita', creiamo tre vettori di dati a partire dall'oggetto letto dal file. 

In [ ]:
z = data[:,0] 
# La scrittura in alto significa che stiamo prendendo 
# tutte le righe della matrice (il simbolo :), 
# ma solo della prima colonna (0)

Md = data[:,1]     #Stessa cosa qui, prendiamo solo la seconda colonna
err_Md = data[:,2] #Stessa cosa qui, prendiamo solo la terza colonna

# Facciamo un grafico

In [ ]:
# utilizziamo la funzione plot appartenente al modulo matplotlib.pyplot (plt) per visualizzare i dati precedentemente letti
# sulle x riportiamo il redshift, prima colonna (ricordiamo che python inizia a indicizzare da zero!)
# sulle y riportiamo il modulo di distanza, seconda colonna.

plt.plot(z, Md,'.')                     #Stiamo chiedendo di mettere z sulle x, Md sulle y, e di rappresentare i punti con '.'
plt.xlabel('Redshift z')                #Stiamo chiedendo di scrivere 'Redshift z' come etichetta dell'asse x
plt.ylabel('M-m')                       #Stessa cosa per l'asse y
plt.title('Dati senza barre d\'errore') #Chiediamo di mettere un titolo al grafico

plt.show()                              # Vediamo il grafico!

In [ ]:
#aggiungiamo ora le barre di errore!
plt.errorbar(z, Md, err_Md, marker='.', linestyle='') 
#Questa funzione e' un po' diversa da plot
#Ci permette di aggiungere una barra d'errore ad ogni punto
#L'opzione linestyle ci permette di chiedere come disegnare una linea che unisca tutti i punti.
#Nel nostro caso, stiamo chiedendo di non usare alcuna linea

plt.xlabel('Redshift z')
plt.ylabel('M-m')
plt.title('Dati con barre d\'errore')

plt.show() 

# Calcoliamo la distanza delle supernovae e il relativo errore

Abbiamo visto durante l'introduzione che la grandezza osservata e' la luminosita' (magnitudine) delle supernovae. Noi vorremmo costruire un diagramma di Hubble che abbia il redshift lungo x e la distanza delle supernovae lungo y. Dunque, dobbiamo ricavare la distanza dell'oggetto a partire dal modulo di distanza. Fortunatamente, le supernovae sono candele standard, per cui conosciamo il modo di legare la luminosita' apparente (osservata) alla loro luminosita' propria e, da quest'ultima, risalire alla distanza!

Convertiamo il modulo di distanza M -m in distanza di luminosita' d e calcoliamo l'errore con la formula di propagazione degli errori. 
Nota che l'unita' di misura della distanza e' il Mpc (mega-parsec), ovvero un milione di parsec. Il parsec corrisponde alla distanza dalla quale una lunghezza pari alla distanza Terra-Sole sottende un angolo ampio un secondo d'arco (1/3600 gradi). In pratica, il parsec e' un cateto di un triangolo rettangolo che ha per altro cateto la distanza Terra-Sole. Da questa definizione, si puo' trovare che il parsec e' circa 1000 miliardi di km. Le supernovae sono davvero oggetti molto distanti!

La colonna 1 dei nostri dati rappresenta il modulo di distanza M - m (magnitudine apparente meno magnitudine assoluta), un indicatore della distanza in base alla luminosità. Per convertire il modulo di distanza nella vera distanza, si usa la formula: 
\begin{equation*}
M-m = 5*log_{10}(d) - 5
\end{equation*}

che ci dice che la distanza è
\begin{equation*}
d = 10^{[(M-m+5)/5]}
\end{equation*}

La distanza ottenuta con questa formula sarebbe espressa in parsec: per convertire in Megaparsec (1 Mpc = $10^6$ pc) dividiamo per $10^6$.

Per calcolare l'errore su d si usa 

\begin{equation*}
err(d) = err(M-m)*[derivata\ di\ d\ rispetto\ (M-m)]
\end{equation*}
\begin{equation*}
= err(M-m)*log(10)*d\ /\ 5
\end{equation*}

In [ ]:
d = 10**(Md/5.+1)/10**6          #Il simbolo di esponente in python e' '**'
err_d = np.log(10)*d/5.*err_Md

# Primo compito
5 minuti

Graficare la distanza di luminosità in funzione del 
redshift con le barre di errore sulle y

In [ ]:
# Scrivi in questa cella il tuo blocco di codice
# ...

In [ ]:
#@title
# Graficare la distanza di luminosità in funzione del 
# redshift con le barre di errore sulle y

plt.errorbar(data[:,0],d,err_d,marker='.',linestyle='')
plt.xlabel('Redshift')
plt.ylabel('Luminosity distance [Mpc]')
plt.title('Gold catalogue')

plt.show()

Come vediamo dal grafico, il catalogo di supernovae nel file contiene oggetti anche molto distanti (z~1.75). Noi sappiamo pero' che la legge di Hubble, cioe' la relazione di proporzionalita' diretta tra la distanza e il redshift, vale strettamente solo per oggetti vicini (z<<1).
Tagliamo quindi il nostro cataologo e consideriamo solo le supernovae con un redshift inferiore ad una soglia scelta, ad esempio z_soglia=0.1.

In [ ]:
# Indichiamo con zcut la soglia oltre la quale non consideriamo più i dati
zcut = 0.1

# la funzione "where" (del modulo numpy) ci permette di selezionare da un vettore la posizione (zlowindex) dei valori 
# (in questo caso z) che soddisfano la condizione voluta
zlowindex=np.where(z<zcut)[0]
zlow=z[zlowindex]
# il vettore zlow contiene solo valori di redshift inferiori alla soglia scelta

# Secondo compito:
5 minuti


Prova a farti scrivere il vettore zlowindex 
 e poi prova a farti scrivere gli elementi del vettore z che corrispondono agli elementi di zlowindex
 Controlla che i redshift che vengono stampati siano effettivamente piu' piccoli di 0.1.

In [ ]:
# Scrivi in questa cella il tuo blocco di codice
# ...

In [ ]:
#@title
print('Gli elementi del vettore zlowindex sono: ')
print(zlowindex)
print('I redshift che corrispondono alle posizioni in zlowindex sono: ')
print(z[zlowindex])

# Terzo compito:
5 minuti

Graficare la distanza di luminosità d in funzione del redshift per i soli valori selezionati 
con le barre di errore sulle y.

In [ ]:
# Scrivi in questa cella il tuo blocco di codice
# ...

In [ ]:
#@title
plt.errorbar(zlow,d[zlowindex],err_d[zlowindex],
             ls='',marker='o')
plt.xlabel('Redshift')
plt.ylabel('Distance [Mpc]')
plt.title('Gold catalogue, z<zcut')

# Stima della costante di Hubble
10 minuti

Proviamo a trovare una relazione che descrive il comportamento dei dati nella figura sopra (questa operazione è comunemente indicata con il termine inglese "fit"). Da questa relazione possiamo ricavare la costante di Hubble!

<figure>
<center>
<img src='https://drive.google.com/uc?export=view&id=12Y3UYFnDHAnMs09Z_CACz06RgPfbJN4W' width="500">
</center>
</figure>

Ad esempio, definiamo due funzioni di fit: una funzione lineare (una retta)
\begin{equation*}
y = mx + q
\end{equation*}

<figure>
<center>
<img src='https://drive.google.com/uc?export=view&id=1Dg9Ch21e1qG5dq9p8N5vpFWvVzApa1Ls' width="300" height="300">
</center>
</figure>

e una funzione quadratica (una parabola)
\begin{equation*}
y = ax^2 + bx + c
\end{equation*}

<figure>
<center>
<img src='https://drive.google.com/uc?export=view&id=1u-kclFKCu_rhFjSJZuHjKICjxSLpxST4' width="400" height="400">
</center>
</figure>

I coefficienti della retta (m, q) e della parabola (a, b, c) sono parametri liberi. Quando facciamo un fit, troviamo i valori di questi parametri liberi in modo che la funzione meglio descriva i dati.

In [ ]:
from scipy.optimize import curve_fit 
#scipy.optimize e' un pacchetto che consente di analizzare dati
#curve_fit e' una funzione che permette di ricavare il valore dei parametri quando si fa un fit,
#dato un insieme di dati e una funzione di fit

#La scrittura in basso e' il modo in cui in python si definiscono delle funzioni
#def funzione(x,a,b,c,d): 
#significa che sto definendo una funzione di x che avra' dei parametri liberi a,b,c,d

#quando dovro' usare questa funzione, potro' invocarla scrivendo
#funzione(xin,ain,bin,cin,din)
#dove xin sono i valori di x in cui voglio che sia calcolata funzione
#e ain,bin,cin,din sono i valori dei parametri che voglio siano usati

#return mi restituisce i valori della funzione

# Prima funzione: fit lineare ---> riconosci l'equazione di una retta con pendenza m=300000/H0 e intercetta q=0?
def line(x, H0): 
    q=0
    speed_of_light = 300000.
    return speed_of_light/H0 * x + q

# Seconda funzione: fit quadratico ---> riconosci l'equazione di una parabola? Cosa c'è al posto dei parametri a,b,c?
def parabola(x, H0, q0): 
    c=0
    speed_of_light= 300000.
    return speed_of_light/H0*(1.-q0)*0.5*x**2 + speed_of_light/H0*x + c


Iniziamo con un fit lineare. Ricorda che $d=z*c\ /H_0$. Consideriamo il dataset contenente solo le supernovae con redshift z < 0.1 (zlow).

In [ ]:
# La funzione "curve_fit" richiede in input la funzione scelta per il fit (in questo caso line), 
# le coordinate x e y dei dati (zlow, d) e l'errore sui dati sigma (err_d);
# curve_fit restituisce come output i parametri della funzione usata per il fit e il relativo errore.
# Nel nostro caso il parametro cercato è il denominatore
# della pendenza della retta, che chiamiamo H0 (costante di Hubble al tempo attuale)

par, par_err = curve_fit(line, zlow, d[zlowindex], sigma=err_d[zlowindex])

# usiamo la funzione "print" per vedere i risultati ottenuti
print('H0 = (',par[0],'\u00B1',np.sqrt(par_err[0,0]), ') km/s/Mpc')

# Nota due cose:
# 1) i parametri sono restituiti in un vettore, mentre gli errori in una matrice (se ti stai chiedendo perche',
#    chiedilo a noi!)
# 2) gli errori sui parametri restituiti da curve_fit sono al quadrato, quindi dobbiamo fare la radice con np.sqrt(...)

Abbiamo ottenuto la nostra stima della costante di Hubble!
Il risultato ci dice che l'Universo sta espandendo in maniera tale che oggi un oggetto distante 1 Mpc da noi si allontana con una velocita' di 64.2 km/s. Non male... In un secondo, viene quasi coperta la distanza Ferrara-Padova!

Plottiamo la funzione di fit ottenuta e i dati

In [ ]:
# dati
plt.errorbar(zlow, d[zlowindex], err_d[zlowindex], ls='', marker='o')

# definiamo il dominio della funzione di fit, cioè dove voglio calcolarla per il grafico
zz = np.arange(0, 0.10, 0.01) 
# La funzione "arange(x,y,step)" (del modulo numpy) produce un vettore contenente tutti i valori da x a y-step con passo=step.

# calcoliamo la funzione del fit lineare usando il parametro appena calcolato (H0=par[0]).
speed_of_light=300000.
fit_lineare = zz*speed_of_light/par[0]
plt.plot(zz, fit_lineare, ls='-')
plt.xlabel('Redshift')
plt.ylabel('Distance [Mpc]')
plt.title('Gold catalogue, z<0.1')

plt.show()

Sembra proprio che questa curva descriva bene l'andamento dei punti che abbiamo selezionato.

A questo punto, vogliamo sapere cosa succede se invece proviamo a considerare oggetti piu' lontani, come fecero i due gruppi di High-z Supernova Search Team e Supernova Cosmology Project nei primi anno '90 (vinsero il premio nobel grazie a questo lavoro!!!).

Consideriamo quindi un nuovo sottoinsieme di dati. Selezioniamo tutte le supernovae con redshift moderatamente alto, ad esempio z<0.6.

# Stima del parametro di decelerazione q0

# Quarto compito:
10 minuti

Con un procedimento analogo al caso z<0.1 seleziona gli oggetti con redshift minore di 0.6.  Poi, esegui un fit lineare e grafica la funzione risultante sopra i dati

In [ ]:
# Scrivi in questa cella il tuo blocco di codice
# ...

In [ ]:
#@title

zcut=0.6
zlowindex=np.where(z<zcut)[0]
zlow=z[zlowindex]
popt, popt_err = curve_fit(line, zlow, d[zlowindex], 
                       sigma=err_d[zlowindex])
print('H0 = (',popt[0],'\u00B1',np.sqrt(popt_err[0,0]), ') km/s/Mpc')
plt.errorbar(zlow,d[zlowindex],err_d[zlowindex],
             ls='',marker='o')
zz=np.arange(zcut*100)/100.
plt.plot(zz,zz*300000/par[0],ls='-',color='C1')
plt.xlabel('Redshift')
plt.ylabel('Distance [Mpc]')
plt.title('Gold catalogue, z<0.6')

plt.show()

Cosa ne pensi del fit? Riproduce bene l'andamento dei dati, soprattuto ad "alti" redshift?

Proviamo con un fit quadratico!

# Quinto compito:
10 minuti

Con un procedimento analogo al caso z<0.1, seleziona gli oggetti con redshift minore di 0.6.

Esegui un fit quadratico e grafica la funzione risultante sopra i dati.

Ricorda che in questo caso i parametri della funzione da determinare sono due: H0 (la costante di Hubble) e q0 (il parametro di decelerazione).

L'ordine con cui vengono restituiti da curve_fit corrisponde all'ordine in cui compaiono nella definizione della funzione parabola. Nel nostro caso abbiamo prima H0 e poi q0. Quindi, se chiami il vettore dei parametri 'par2', avrai
\begin{equation*}
H_0 = par2[0], q_0 = par2[1].
\end{equation*}

L'errore su ciascun parametro i corrisponde alla radice quadrata dell'elemento ii della matrice degli errori restituita da curve_fit. Quindi, se chiami la matrice degli errori 'par_err2', l'errore su H0 sarà
\begin{equation*}
\sqrt{par\_err2[0,0]}
\end{equation*}

e quello su q0 sarà
\begin{equation*}
\sqrt{par\_err2[1,1]}
\end{equation*}

Alla fine confronta il risultato con il plot precedente.

In [ ]:
# Scrivi in questa cella il tuo blocco di codice
# ...

In [ ]:
#@title

zcut=0.6
zlowindex=np.where(z<zcut)[0]
zlow=z[zlowindex]

par2, par_err2 = curve_fit(parabola, zlow, d[zlowindex], 
                       sigma=err_d[zlowindex])
print('H0 = (',par2[0],'\u00B1',np.sqrt(par_err2[0,0]), ') km/s/Mpc')
print('q0 = ',par2[1],'\u00B1',np.sqrt(par_err2[1,1]))

plt.errorbar(zlow,d[zlowindex],err_d[zlowindex],
             ls='',marker='o')

zz=np.arange(zcut*100)/100.
plt.plot(zz,zz*300000/par[0],ls='-',label='linear',color='C1')
plt.plot(zz,zz*300000/par2[0]+
         zz**2*300000/par2[0]*0.5*(1-par2[1]),
         ls='-',label='quadratic',color='C2')

plt.xlabel('Redshift')
plt.ylabel('Distance [Mpc]')
plt.title('Gold catalogue, z<0.6')
plt.legend()

plt.show()

Decisamente meglio! Le supernovae distanti non seguono proprio una retta! Hai notato che valore di bestfit ottieni per q0? Che interpretazione puoi dare a questo risultato? L'Universo e' in espansione? Se si', decelerata o accelerata?

Ricordando quanto detto durante l'introduzione, ovvero che q0 puo' essere scritto come funzione delle componenti che costituiscono l'Universo, proviamo a vedere quanta materia e quanta energia oscura possiamo aspettarci dato il valore di q0 appena misurato.

Riportiamo la formula per comodita':
\begin{equation*}
q_0 = \Omega_m\ /\ 2 - \Omega_\Lambda
\end{equation*}

dove $\Omega_m$ e $\Omega_\Lambda$ indicano la quantità di materia ed energia oscura nell'Universo.

# Coffee break!

# Stima delle quantita' di materia ed energia oscura

Per prepararci a fare questo passo finale, impariamo prima ad usare nuovi costrutti: for e if 

In [ ]:
vec = np.arange(7,27,1)
print("vettore iniziale:",vec)
# Utilizziamo "for" per scorrere fra gli elementi del vettore vec.
# Selezioniamo con "if" tutti i numeri compresi tra 13 e 17.

min_value = 13
max_value = 17
new_vec = list() #Questo comando crea un oggetto vuoto (una lista, in particolare), che possiamo riempire in seguito
for value in vec: 
    print("L'elemento del vettore che stiamo esaminando è", value)  
    if(value > min_value and value < max_value):
        new_vec.append(value) #Questo comando 'appende' l'elemento value alla fine della lista new_vec

# Stampiamo il nuovo vettore!
print("elementi selezionati:",new_vec)

# Sesto compito:
15 minuti

Definiamo:

1) om = $\Omega_m$, cioè la quantita' di materia nel nostro universo

2) ol = $\Omega_\Lambda$, cioè la quantita' di energia oscura nel nostro universo

Entrambi questi parametri, per come sono definiti, possono assumere solo valori non negativi.

Sapendo che $q_0 = \Omega_m\ /\ 2 - \Omega_\Lambda$, calcola le coppie di punti (om;ol) che restituiscono un valore di q0 in accordo con i dati.

In altre parole, se abbiamo stimato q0_est con un errore q0_err, vogliamo i valori (om;ol) che soddisfano
\begin{equation*}
q0\_est - q0\_err < q0 < q0\_est + q0\_err
\end{equation*}

In [ ]:
q0_est = par2[1]
q0_err = np.sqrt(par_err2[1,1])
accepted = list()

# Facciamo variare om e ol tra 0 e 1 a step di 0.01. 
# I parametri sono due --> servono due cicli "for" per coprire tutte le possibili combinazioni (om, ol)
# Completa il blocco di codice in basso:

for om in ------ :  
    for ol in ------ :
        q0 = ...
        # Selezione con il costrutto "if" tutti i valori di q0 compresi tra q0_est + err e q0_est - err
        if(------):
            accepted.append([om,ol])

In [ ]:
#@title
q0_est = par2[1]
q0_err = np.sqrt(par_err2[1,1])
accepted = list()
for om in np.arange(0,1,step=0.01):  
    for ol in np.arange(1,0,step=-0.01):
        q0 = om/2. - ol
        if(q0>q0_est-q0_err and q0<q0_est+q0_err):
            accepted.append([om,ol])

# Settimo compito:
15 minuti

Adesso creiamo un grafico con i valori selezionati

In [ ]:
# Prima creiamo due elenchi separati per i valori di Omegam e OmegaL
om_ol_vec = np.array(accepted)
om_vec = om_ol_vec[:,0]
ol_vec = om_ol_vec[:,1]

# La funzione "scatter" (dal modulo plt) permette di graficare dei punti a partire dalle coordinate
plt.scatter(om_vec, ol_vec)

# Calcoliamo Omega_lambda (ol) invertendo la relazione q0 = om/2-ol, e utilizzando Omega_materia appena calcolato (om_vec)
# Completa le linee di codice in basso:
ol_vec_calc = ...

# Aggiungiamo al nostro scatter-plot la retta cosi' ottenuta in funzione di om_vec
plt.plot(...)

# Aggiungiamo il singolo punto che rappresenta il valore di Omega_materia e Omega_lambda 
# stimati da misure dei più recenti esperimenti cosmologici.
plt.scatter(0.317,0.683)
plt.xlabel('Omega_m')
plt.ylabel('Omega_L')

plt.show()

In [ ]:
#@title
om_ol_vec = np.array(accepted)
om_vec = om_ol_vec[:,0]
ol_vec = om_ol_vec[:,1]

plt.scatter(om_vec, ol_vec)
ol_vec_calc = om_vec/2.-q0_est
plt.plot(om_vec, ol_vec_calc, '-r')
plt.scatter(0.317,0.683)
plt.xlabel(r'$\Omega_m$')
plt.ylabel(r'$\Omega_\Lambda$')

plt.show()

Ottimo! Hai appena ottenuto la regione nel piano  $(\Omega_m;\Omega_\Lambda)$ che restituisce valori di q0 in accordo con i dati!
Questo significa che tutti i punti dentro questa regione descrivono in maniera plausibile il nostro Universo, secondo i dati di supernovae usati.

Nota che il valore $\Omega_\Lambda = 0$ non e' incluso in questa regione. Significa che un Universo senza energia oscura e' escluso dai dati di supernovae! 

Il puntino arancione nel grafico corrisponde ai valori di $\Omega_m$ e $\Omega_\Lambda$ ottenuti dalla combinazione di misure da diversi esperimenti cosmologici (ad esempio, misure di radiazione di fondo cosmico o misure di abbondanza di materia nell'Universo, chiedici di piu' se l'argomento ti interessa!). Come vedi, il puntino cade nella regione preferita dalle supernovae! Questo significa che diversi esperimenti cosmologici sono in accordo tra loro sul fatto che il nostro Universo e' costituito prevalentemente da energia oscura. 

E qui inizia un nuovo viaggio... perche', benche' l'energia oscura rappresenti il 70% di tutto quello che c'e' nell'Universo, ancora non sappiamo cosa essa sia davvero. In gergo scientifico, si dice che l'energia oscura fornisce una descrizione soddisfacente dal punto di vista fenomenologico (dal punto di vista, cioe', della descrizione di un fenomeno, in questo caso l'espansione accelerata), ma per nulla soddisfacente dal punto di vista della fisica fondamentale (ovvero, dal punto di vista di quali siano le proprieta' specifiche dell'energia oscura: che cos'e'? Da cosa deriva? La sua densita' e' costante nel tempo o varia nel tempo?). Questo e' uno dei campi di ricerca piu' attivi nella cosmologia sia teorica che osservativa moderna.

Ti interessa saperne di piu', su questo o su altri argomenti di cosmologia? Contattaci! 
Ti interessa al punto tale che ti piacerebbe seguire un percorso di studi che ti porti a lavorare in questo campo? Contattaci!
Hai dei dubbi o ti e' poco chiaro quello che abbiamo fatto oggi? Contattaci!